elements/lisk-cryptography/test/ed.spec.ts

```diff
@@ -52,10 +56,20 @@ ${signature.toString('hex')}
 `.trim();
 
 	describe('#signMessageWithPrivateKey', () => {
-		it('should create a signed message using a private key', () => {
+		it('should create a signed message using a private key when message is in string format', () => {
 			const signedMessage = signMessageWithPrivateKey(message, privateKey);
 			expect(signedMessage).toMatchSnapshot();
 		});
+
+		it('should create a signed message using a private key when message is in Buffer format', () => {
+			const signedMessage = signMessageWithPrivateKey(messageBuffer, privateKey, tag);
+			expect(signedMessage).toMatchSnapshot();
```

Comment:
Not a big deal, but I was a bit confused in the beginning because I expected that the signatures must be equal in these two cases...but they are not due the the tag in the second case. As the custom tag is tested in the following test, shouldn't these two cases use the same (default) tag and and the expectation should be that the two signatures are equal? I would find it a bit nicer to define the expected signature as a constant in the test set-up and to have in both tests the expectation that the result is equal to this single constant, instead of two individual snapshots where one must manually check that they are identical.

Reply:
> As the custom tag is tested in the following test, shouldn't these two cases use the same (default) tag and and the expectation should be that the two signatures are equal?

Good point about the tag. Probably a copy/paste error 😅
I removed it from the test case which checks general validity.

> I would find it a bit nicer to define the expected signature as a constant in the test set-up and to have in both tests the expectation that the result is equal to this single constant, instead of two individual snapshots where one must manually check that they are identical.

I don't think it makes that much sense to create an expected signature constant, because we would obtain it anyway using `signMessageWithPrivateKey()`. It means that the unit test would only check if the 2 invokes of the same function with the same arguments returned the same result... which is not very valuable.

But following your suggestion, I have combined `string` and `Buffer` messages in the same test case, which now also tests that the signatures of equivalent messages in different formats are the same. That is valuable.

Reply:
thanks